sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/UnsupportedOperationsSuite.scala

```diff
@@ -1015,3 +1015,42 @@ class UnsupportedOperationsSuite extends SparkFunSuite with SQLHelper {
     override def nodeName: String = "StreamingRelationV2"
   }
 }
+
+object FlatMapGroupsWithState {
```

Comment:
It's better to use a different class name, such as TestFlatMapGroupsWithState. Otherwise, it's hard to tell which one is being used in a test.

Reply:
done